examples/NBomber.Examples.CSharp/Program.cs

```diff
@@ -15,19 +15,19 @@ static void Main(string[] args)
             //var scenario = HttpScenario.BuildScenario();  
             var scenario = SimplePushScenario.BuildScenario();
 
-            var asserts = new [] {
```

Comment:
Asser.ForScenario looks better than AssertFor
+ I  assume that letter you could add something else for just plain Assert, like Asser.IsTrue, for such case AssertFor will be strange

Reply:
Maybe Assertion.ForScenario ?